WengerK/vscode-highlight-bad-chars
# Add tests

### 💬 Describe the pull request
- Add Jest tests to prevent further regression.
- Add Github Actions tests & styles

### 🔢 To Review
Steps to review the Jest integration:
1. Install new dependencies `npm install`
2. Run tests `npm run test`

Steps to review the Github Actions integration:
1. Check Github Actions run both tests and linter
2. See https://github.com/WengerK/vscode-highlight-bad-chars/actions

### 🔖 Does it need an entry in the CHANGELOG ?
 - [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`
